### ISSUE TITLE
Language and incos not working

### ISSUE THREAD

**Issue by JessicaPuebla** [kartik-v/bootstrap-fileinput#1351]
I've downloaded the last version of the plugin and:

I already set the language property and included the locale, it doesn't work.
The same issue with icons.

$("#img-producto").fileinput({
theme: 'fas',
language: 'es',
});

And I also include in my file

<script src="vendor/fileinput/themes/fas/theme.js"></script> <script src="vendor/fileinput/js/locales/es.js" type="text/javascript"></script>

Help and sorry about my English!

**Comment by kartik-v** [comment]
It seems a problem in the sequence of JS and CSS files you are loading (check docs).  Or you may have missed a file like Fontawesome CSS.  Post the sequence of JS and CSS that is loaded on your page.

**Comment by NeilSiao** [comment]
@JessicaPuebla  I just encounter same problem, u save my day

**Comment by kartik-v** [comment]
Yes suggest to please [read the usage section where this is clearly HIGHLIGHTED](https://plugins.krajee.com/file-input#usage)
